cfht2caom2/data_source.py

```diff
@@ -96,19 +96,19 @@ def __init__(self, config, cadc_client, recursive=True):
         self._recursive = recursive
         self._logger = logging.getLogger(self.__class__.__name__)
 
-    def clean_up(self):
-        self._logger.debug('Begin clean_up')
+    def clean_up(self, entry):
+        self._logger.debug(f'Begin clean_up with {entry}')
```

Comment:
What are the expected types for entry?

Reply:
Added a doc comment to describe the possible types and cases.